requirements-dev.txt

```diff
@@ -0,0 +1,14 @@
+pytest
+pytest-cov
+mock
+pymysql
+pymssql
+psycopg2
+pymssql
+psycopg2
+flake8
+# FIXME: I am not sure these are needed.
```

Comment:
This needs a careful review.
I see packages dependencies all over the place and they don't really seems to be used.
I do know that, like for the `wofpy` case, people expected these to be installed even when the module in question is not using them directly.

If that is the case we should create an integration test env to add and test those and the expected functionality.